lk2nd/target_display_cont_splash_mdp3.c

```diff
@@ -0,0 +1,45 @@
+#include <arch/defines.h>
+#include <bits.h>
+#include <debug.h>
+#include <reg.h>
+#include <string.h>
+#include <target.h>
+
+#include <dev/fbcon.h>
+#include <kernel/event.h>
+#include <kernel/thread.h>
+#include <mdp3.h>
+#include <platform.h>
+#include <platform/clock.h>
+#include <platform/iomap.h>
+#include <platform/timer.h>
+
+
+static struct fbcon_config fb = {
+	.bpp    = 24,
+	.format = FB_FORMAT_RGB888,
+};
+
+static void mdss_mdp_cmd_kickoff(void) {
+	writel(0x00000001, MDP_DMA_P_START);
+	mdelay(10);
+}
+
+int mdp3_read_config(struct fbcon_config *fb)
+{
+	fb->base = (void *) readl(MDP_DMA_P_BUF_ADDR);
+	fb->width = readl(MDP_DMA_P_BUF_Y_STRIDE)/3;
```

Comment:
isn't this supposed to be /4 for a8r8g8b8?

Reply:
also shouldn't the format above be variable? or does it just work :tm:?

Reply:
> isn't this supposed to be /4 for a8r8g8b8?

/3 works fine, but I'm testing it on a Lumia 550, and Lumias used some different format afaik, so I'm not sure what to do here, might be necessary to change it as it was on msm8226 lumia lk2nd.

> also shouldn't the format above be variable? or does it just work tm?

It just works xd, it's the same in target_display_cont_splash_mdp5